screens/EpisodePrediction.tsx

```diff
@@ -13,86 +12,13 @@ import { uploadJSONToFirebase } from "../clients/firebaseInteractor";
 import { View, Text, Button } from "../components/Themed";
 import Colors from "../constants/Colors";
 import { continueButtonStyle } from "../utils/StylingUtils";
-import Contact from "../constants/Contact";
 import { STORAGE_KEYS } from "../utils/AsyncStorageUtils";
+import { useMainNavigation } from "../hooks/useMainNavigation";
 
-function EvenSpacedView() {
+export function EvenSpacedView() {
   return <View style={{ flex: 1 }} />;
 }
```

Comment:
If you're exporting this maybe put it in some common component file

Reply:
yeeet good point